edge-runtime/lib/response.ts

```diff
@@ -1,5 +1,5 @@
 import type { Context } from '@netlify/edge-functions'
-import { HTMLRewriter } from '../vendor/deno.land/x/html_rewriter@v0.1.0-pre.17/index.ts'
+import { HTMLRewriter, init, type TextChunk } from 'https://deno.land/x/htmlrewriter@v1.0.0/src/index.ts'
```

Comment:
@JakeChampion I'm not super familiar with this setup, but it looks like all our deno imports use a relative path and should be included [in this file](https://github.com/opennextjs/opennextjs-netlify/blob/767b812969b8c099a2f60d87e5d0866594ecea18/edge-runtime/vendor.ts#L15) 🤔

Reply:
Should be covered by https://github.com/opennextjs/opennextjs-netlify/pull/2707/commits/4a359c69c79f388cb936d18f4e9d572b588e2061